pom.xml

```diff
@@ -0,0 +1,71 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.nuxeo.ecm.distribution</groupId>
+    <artifactId>nuxeo-distribution</artifactId>
+    <version>8.10</version>
+  </parent>
+
+  <groupId>org.nuxeo.marketplace.mongodb.ext</groupId>
+  <artifactId>parent</artifactId>
+  <packaging>pom</packaging>
+  <name>Nuxeo MongoDB Extension Marketplace Parent</name>
+  <version>1.0.0-SNAPSHOT</version>
+
+  <modules>
+    <module>marketplace</module>
+  </modules>
```

Comment:
I thought about that, but we need an environment with a MongoDB running somewhere, as we need this addons only for 8.10, maybe to much work to make Jenkins job work in a MultiDB environment as this code will be merged in Nuxeo Platform for 9.10.